App/src/main/java/vut/fit/gja2023/app/util/CsvReaderConfig.java

```diff
@@ -0,0 +1,22 @@
+package vut.fit.gja2023.app.util;
+
+import lombok.Data;
+
+@Data
+public class CsvReaderConfig {
+    private final int[] columnOrder;
+    private final boolean havingHeader;
+    private final char separator;
+    
+    public CsvReaderConfig(int[] columnOrder, boolean havingHeader) {
+        this.columnOrder = columnOrder;
+        this.havingHeader = havingHeader;
+        this.separator = ',';
+    }
+    
+    public CsvReaderConfig(int[] columnOrder, boolean havingHeader, char separator) {
+        this.columnOrder = columnOrder;
+        this.havingHeader = havingHeader;
+        this.separator = separator;
+    }
```

Comment:
ne ako neviem čo ti tam nefungovalo, proste kedze je to private final, tak tam moze byt aj `@RequiredArgsConstructor`, ale aj `@AllArgsConstructor`.... Ako hej samozrejme fungovat bude fungovat aj teraz, len Lombok rieši prave tento boilerplate kód, tak je dobre ho pouzivat

Reply:
este posledny koment v novych upravach...